src/qt/optionsdialog.cpp

```diff
@@ -97,17 +97,43 @@ OptionsDialog::OptionsDialog(QWidget *parent, bool enableWallet) :
     for (const QString &langStr : translations.entryList())
     {
         QLocale locale(langStr);
+        QString langName = locale.nativeLanguageName();
+        langName[0] = langName[0].toTitleCase(); /* Converts applicable first characters to uppercase */
 
-        /** check if the locale name consists of 2 parts (language_country) */
-        if(langStr.contains("_"))
+        if(locale.nativeLanguageName().length()==0) /* If locale name is an empty string, fix it manually. */
         {
-            /** display language strings as "native language - native country (locale name)", e.g. "Deutsch - Deutschland (de)" */
-            ui->lang->addItem(locale.nativeLanguageName() + QString(" - ") + locale.nativeCountryName() + QString(" (") + langStr + QString(")"), QVariant(langStr));
+            if(langStr.contains("szl"))
```

Comment:
Take a look at the coding guidelines please

Reply:
Where could I find that?

Reply:
@GChuf: I would guess @emilengler is referring to https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md

Reply:
@jonatack thank you. I did a quick search but was not able to find where the guidelines are.
@emilengler I've since updated the file.